src/applications/static-pages/dependency-verification/components/dependencyVerificationModal.jsx

```diff
@@ -11,6 +11,30 @@ import DependencyVerificationList from './dependencyVerificationList';
 import DependencyVerificationFooter from './dependencyVerificationFooter';
 
 const DependencyVerificationModal = props => {
+  const targetNode = document.getElementsByTagName('body');
+  const [nodeToWatch] = useState(targetNode[0]);
```

Comment:
can this just be `const targetNode = document.body;` or `const targetNode = document.getElementsByTagName('body')[0];` since the state never gets updated?